examples/platforms/nrf52840/spi-slave.c

```diff
@@ -0,0 +1,233 @@
+/*
+ *  Copyright (c) 2018, Nordic Semiconductor, Inc.
```

Comment:
nit: anything outside `third_party` should be "The OpenThread Authors".

Reply:
Sure, I will fix that.